src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -180,17 +194,20 @@ protected function commandsNamed(array $names)
      */
     protected function addImpliedCommands(Connection $connection, Grammar $grammar)
     {
-        if (count($this->getAddedColumns()) > 0 && ! $this->creating()) {
-            array_unshift($this->commands, $this->createCommand('add'));
-        }
-
-        if (count($this->getChangedColumns()) > 0 && ! $this->creating()) {
-            array_unshift($this->commands, $this->createCommand('change'));
-        }
-
```

Comment:
Step 2. We have already added `ColumnDefinition`s to list of Blueprint's commands, so there is no need to prepend a command for added / changed columns.